### PR TITLE
Include a section in README about setting up developer environment.

### PR DESCRIPTION
**Pull request by facundobatista**
Also improved a detail in a test because newer pylint complains.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,21 @@ The library provides two main functionalities:
...
 
+# Setting up the environment
+
+Install at system level:
+
+    sudo snap install pyright
+
+Create a virtual environment, activate it, and install developer dependencies:
+
+    python3 -m venv env
```
Comment (cmatsuoka) [inline README.md:26]: Should we use a user-selectable name such as `python3 -m venv <dir name>`?

**Review by cmatsuoka — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -15,6 +15,21 @@ The library provides two main functionalities:
...
+    source env/bin/activate
+    pip install .[dev]
+
+That's all.
+
+
```
Comment (jugmac00) [inline README.md:32]: Just a note.

You could also setup the development environment via

```
tox --devenv <dir name>
```

But right - I just saw you install `tox` the other way round with `.[dev]` - so my suggestion does not make much sense for this setup :-)

File: README.md (hunk 1 above)
Comment (facundobatista) [reply README.md:26]: In this case, I prefer to give instructions that you can copy and execute. Not that you need to understand how to modify to change them.

File: README.md (hunk 2 above)
Comment (facundobatista) [reply README.md:32]: Yes, there are multiple ways. We just need to indicate one so we enable newcomers to start easily.

**Pull request merged**